spec/models/content_item_spec.rb

```diff
@@ -1,124 +1,75 @@
 require 'rails_helper'
 
 RSpec.describe ContentItem do
-  let(:content_item) do
-    ContentItem.new(
-      content_item_params
-    ).tap do |content_item|
-      content_item.link_set = Tagging::ContentItemExpandedLinks.new(link_set_params)
-    end
-  end
+  subject(:content_item) { described_class.new(data, blacklist: blacklist) }
```

Comment:
Hmm, not sure about this. We're not using `subject` other specs in this app, and I'd probably prefer be consistent (but on the hand these specs weren't great to begin with, so I'd like to refactor it some day anyway).